### DISCUSSION TITLE
Как отличать закрытые дискуссии от открытых?

### DISCUSSION THREAD

**Discussion by nergal-perm** [nergal-perm/tg-core#32] [Вопросы] [answered]
Github не позволяет формально "закрывать" дискуссии, поэтому в общем списке дискуссий непонятно, каков статус по ним.

Варианты решения, см. ниже в комментариях.

**Comment by nergal-perm** [comment]
После получения результата обсуждения изменять заголовок дискуссии на что-то вроде `[X] Полностью удаленная работа`, т.е. добавлять префикс `[X]` или любой другой, по которому мы будем понимать, что дискуссия закрыта.

**Comment by nergal-perm** [comment, marked as answer]
Дискуссии могут быть двух типов - "форум" и "вопрос". 

Форумы не имеют состояний, они по сути бесконечны, и сейчас категория дискуссий "Идеи" - это именно формат форума.

Для вопросов есть состояния "без ответа" и "с ответом" - можно использовать именно этот тип дискуссии, подводить ее итог в последнем комментарии, помечать его как принятый ответ на начальный вопрос, и таким образом понимать, что дискуссия закрыта.

Дискуссии с принятым ответом можно отфильтровать на главной странице с общим списком дискуссий проекта.

**Comment by nergal-perm** [comment]
Создать новую категорию дискуссий (типа "Закрытые обсуждения") и после окончания обсуждения переносить его в эту категорию. Возможно, новую категорию стоит сделать типа "Вопрос-ответ", чтобы явно помечать дискуссии в категории "отвеченными" и тогда их можно будет отфильтровать на главной странице со общим списком обсуждаемых в проекте тем.

**Reply by citerm** [reply]
Мне кажется, что варианты
- с изменением имени дискуссии на [x]... 
- фиксированием итога дискуссии и пометки этого ответа в виде ответа на главный вопрос
- перенос дискуссии в новую категорию закрытых
подходят для однозначного разделения открытых впоросов и завершенных, думаю стоит использовать все перечисленные тобой способы.

**Reply by nergal-perm** [reply]
А если оттолкнуться от варианта использования? 

Я бы хотел на домашней странице Дискуссий иметь возможность увидеть только активные, т.е. те, по которым еще идет обсуждение и они требуют внимания. При этом они могут находиться в разных категориях: Вопросы, Идеи или Задачи. Github, конечно, может применить фильтр (Категория ИЛИ Категория ИЛИ Категория), но хотелось бы не руками писать этот фильтр, а применить его одним кликом. 

Однокликовая возможность одна - это кнопка "Unanswered", но она сработает только, если категория Дискуссий была "Вопрос". Так что я бы склонился к использованию **только** варианта с пометкой комментария в виде ответа на главный вопрос.

Что скажешь? Попробуем?

**Reply by citerm** [reply]
Ну давай эту дискуссию сведем к какому то выводу, зафиксируй комментарий с итоговым решением как ответ на главный вопрос. Я за то, чтобы попробовать. Меня пока больше интересует внутреннее содержание дискуссии и то, что она должна вылиться в какое то описание в вики. Остальное на твой вкус))
